### DISCUSSION TITLE
Could NOT find OpenMP_C

### DISCUSSION THREAD

**Discussion by 123xyl** [ispc/ispc#2471] [Q&A] [answered]
![Uploading image.png…]()

**Comment by aneshlya** [comment]
The image is not loading.
ISPC Runtime depends on OpenMP Runtime. Consult your Linux distribution documentation for the installation of OpenMP runtime instructions. No specific version is required. For example on Ubuntu: `apt-get install libomp-12-dev`.
In the future version of ISPC the dependency to OpenMP is planned to be removed.

**Reply by 123xyl** [reply]
how to install on centos

**Reply by aneshlya** [reply]
https://centos.pkgs.org/7/centos-sclo-rh-x86_64/llvm-toolset-7-libomp-4.0.1-1.el7.x86_64.rpm.html

**Reply by 123xyl** [reply]
Still hasn't solved the problem

**Reply by aneshlya** [reply, marked as answer]
Starting from ISPC v1.20 ISPC Runtime doesn't have dependency to OpenMP, but has dependency to TBB.
Please use latest ISPC version and try to install TBB using
```
dnf -y update && dnf install -y tbb-devel
```
